client/components/cart.js

```diff
@@ -17,15 +17,20 @@ export class Cart extends Component {
     const { handleSubtract } = this.props;
     const { handleAdd } = this.props;
 
+    let total = 0;
+
     productIds.forEach(id => {
       this.props.products.filter(product => {
         if (+product.id === +id) {
           product.quantity = this.props.cart[id]
+          total += (product.quantity * product.price)
```

Comment:
We should update the Product model then, so we can access all data in the front end, at the moment we ask for all attributes, except quantity?